change wording against prod to for development (#80)
temporalio/helm-charts: README.md

# Temporal
[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Ftemporalio%2Ftemporal-helm-charts.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Ftemporalio%2Ftemporal-helm-charts?ref=badge_shield)

> Temporal is a distributed, scalable, durable, and highly available orchestration engine to execute asynchronous long-running business logic in a scalable and resilient way. The bundled dependencies included in this Helm chart provide minimal required functionality to ease getting started and experimentation with Temporal. This Helm chart can also be used to install only the Temproal services and configure them to connect to live dependencies.

Reviewer: I would combine the two opener sentences, suggestion:

```suggestion
Temporal is a distributed, scalable, durable, and highly available orchestration engine to execute asynchronous long-running business logic in a scalable and resilient way. This repo contains a basic [Helm](https://helm.sh) chart that installs Temporal to a Kubernetes cluster. The dependencies that are bundled with this installation solution offer an easy way to **experiment** with the Temporal server. This Helm chart can also be used to install just the Temporal server and configure it to connect to live dependencies.
```

# Install Temporal service on a Kubernetes cluster

## Prerequisites

This sequence assumes that your system is configured to access a kubernetes cluster (e. g. [AWS EKS](https://aws.amazon.com/eks/), [kind](https://kind.sigs.k8s.io/), or [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)), and that your machine has [AWS CLI V2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html), [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and [Helm v3.1.x](https://helm.sh) installed and able to access your cluster.

Download Helm dependencies:

```bash
~/temporal-helm$ helm dependencies update
```

## Install Temporal Helm chart

Temporal can be configured to run with a variety of different dependencies and the Helm chart installs these by default:

* Cassandra
* ElasticSearch
* Kafka (with Zookeeper)
* Promethueus
* Grafana

MySQL can be swapped in for Cassandra but is not deployed as part of this Helm chart.

> The following sections will walk through installing with everything included on a single node cluster and conclude with a configurtion that lets you bring all of your own dependencies.

Reviewer: ```suggestion
The following sections walk through the server installation on a single node cluster using all of the included dependencies, and concludes with an installation configuration that lets you bring all of your own dependencies.
```

> ### Install minimal versions of only requried dependencies

Reviewer: Ya, I like this direction better. But I suggest rephrasing it to:
```suggestion
### Minimal installation with required dependencies only
```

However, not sure if that loses the intended meaning - as this sentence "Install minimal versions of only requried dependencies" could very easily be suggesting that minimal versions is referring to the dependency versions. So just check me on that -

### Install Temporal

Temporal can be configured to run with a variety of different dependencies. By default this Helm chart deploys Cassandra, ElasticSearch, Kafka, Zookeeper, Promethueus and Grafana to support various features of Temporal. Cassandra can be swapped for MySQL if desired. The following sections will walk through deploying with everything included on a single node cluster and conclude with a configurtion that lets you bring all of your own dependencies.

> #### [Tiny] Batteries Included for Getting Started / Development

Reviewer: What does [Tiny] mean?
I don't like "Batteries Included" because it begs too many questions. Try using a heading that doesn't use an analogy in this case.

```

> ### Install with your own ElasticSearch

Reviewer: Nice. Ya, BYOX is fun for labeling parties - but, I think this is better for our situation 👍

~/temporal-helm$ helm install -f values/values.cassandra.yaml temporaltest . --timeout 900s
```

#### Bring Your Own Everything: Just Deploy Temporal

> In a production environment, where all dependencies are already hardened and operational, it's possible to deploy only the Temporal server components with this Helm chart. Whether using MySQL or Cassandra, the preceeding steps to configure schema should be followed. This example assumes Cassandra, demonstrates setting values on the commandline rather than via environment, and enabled TLS for the database connection:

Author: ```suggestion
In a production environment, where all dependencies are already hardened and operational, it's possible to deploy only the Temporal server components with this Helm chart. Whether using MySQL or Cassandra, the preceeding steps to configure schema should be followed. This example assumes Cassandra, demonstrates setting values on the commandline rather than via environment, and enables TLS for the database connection:
```

> In a production environment, where all dependencies are already hardened and operational, it's possible to deploy only the Temporal server components with this Helm chart. Whether using MySQL or Cassandra, the preceeding steps to configure schema should be followed. This example assumes Cassandra, demonstrates setting values on the commandline rather than via environment, and enables TLS for the database connection:

Reviewer: Also consider using "live" in place of "production" - i.e. a "In a live environment..."

~/temporal-helm$ helm install -f values/values.cassandra.yaml temporaltest . --timeout 900s
```

### Install and configure Temporal

> In a live environment, where all dependencies are already hardened and operational, it's possible to install only the Temporal server components with this Helm chart. Whether using MySQL or Cassandra, the preceeding steps to configure schema should be followed. This example assumes Cassandra, demonstrates setting values on the commandline rather than via environment, and enables TLS for the database connection:

Reviewer: This line is a bit confusing:
> "In a live environment, where all dependencies are already hardened and operational, it's possible to install only the Temporal server components with this Helm chart"

It sounds like we are trying to get the point across that, IF an application environment already uses certain systems that Temporal uses as dependencies, then those systems can continue to be used, and this Helm chart can install the minimal pieces of Temporal such that Temporal can then be configured to use those systems as its dependencies ?

Author: yes... do you have a better/shorter way to say all that?

Reviewer: What about:

```suggestion
If a live application environment already uses systems that Temporal can use as dependencies, then those systems can continue to be used. This Helm chart can install the minimal pieces of Temporal such that it can then be configured to use those systems as its dependencies.

The example below demonstrates a few things:
1. How to set values via the command line rather than the environment.
2. How to configure a database (shows Cassandra, but MySQL works the same way)
3. How to enable TLS for the database connection.
```
